src/Appwrite/Platform/Workers/UsageDump.php

```diff
@@ -57,230 +55,235 @@ public function action(Message $message, callable $getProjectDB): void
             throw new Exception('Missing payload');
         }
 
-
-        foreach ($payload['stats'] ?? [] as $stats) {
-            $project = new Document($stats['project'] ?? []);
-
-            /**
-             * End temp bug fallback
-             */
-            $numberOfKeys = !empty($stats['keys']) ? count($stats['keys']) : 0;
-            $receivedAt = $stats['receivedAt'] ?? 'NONE';
-            if ($numberOfKeys === 0) {
-                continue;
-            }
-
-            console::log('['.DateTime::now().'] Id: '.$project->getId(). ' InternalId: '.$project->getInternalId(). ' Db: '.$project->getAttribute('database').' ReceivedAt: '.$receivedAt. ' Keys: '.$numberOfKeys);
-
-            try {
+        try {
+            foreach ($payload['stats'] ?? [] as $stats) {
+                $project = new Document($stats['project'] ?? []);
                 $dbForProject = $getProjectDB($project);
+                $projectDocuments = [];
+                $databaseCache = [];
+                $collectionSizeCache = [];
+
```

Comment:
`console::log('['.DateTime::now().'] Id: '.$project->getId(). ' InternalId: '.$project->getInternalId(). ' Db: '.$project->getAttribute('database').' ReceivedAt: '.$receivedAt. ' Keys: '.$numberOfKeys);`

lets add a log here to indicate the job for a project has been picked up